demisto_sdk/commands/common/git_util.py

```diff
@@ -795,3 +795,12 @@ def get_all_changed_files(
             get_only_current_file_names=True,
         )
         return modified_files.union(added_files).union(renamed_files)
+
+    def _is_file_ignored(self, file_path: str) -> bool:
+        """return wether the file is in .gitignore file or not.
+        Args:
+            file_path (str): the file to check.
+        Returns:
+            bool: True if the file is ignored. Otherwise, return False.
+        """
+        return not self.repo.ignored(file_path)
```

Comment:
are you sure this isnt flipped?

is_file_ignored returns not ignored(file)

Reply:
you're right.